src/ngAria/aria.js

```diff
@@ -322,7 +333,7 @@ ngAriaModule.directive('ngShow', ['$aria', function($aria) {
   };
 }])
 .directive('ngDisabled', ['$aria', function($aria) {
-  return $aria.$$watchExpr('ngDisabled', 'aria-disabled');
+  return $aria.$$watchExpr('ngDisabled', 'aria-disabled', []);
```

Comment:
`[]` here is a placeholder for forthcoming `ng-disabled` improvements. This PR is purposefully limited.